source/gui/accPropServer.py

```diff
@@ -59,3 +85,24 @@ def GetPropValue(self, pIDString, dwIDStringLen, idProp):
 			log.exception()
 			return VT_EMPTY, 0
 
+	@abstractproperty
+	def properties(self):
+		""" Returns an array of properties that should be handled by this instance.
+		@rtype: L{comtypes.GUID}*n
```

Comment:
What is the *n at the end of this line?

Reply:
The number of items in the array. May be we should allow people to provide a tuple or list, and do de conversion to comtypes array ourselves. That might be less error prone.